.pylintrc

```diff
@@ -5,11 +5,11 @@
 
 # Python code to execute, usually for sys.path manipulation such as
 # pygtk.require().
-init-hook='import sys; sys.path.append("."); sys.path.append("embedded/dd-agent")'
+init-hook='import sys; sys.path.append("."); sys.path.append(os.path.abspath("embedded/dd-agent"))'
 
 # Add files or directories to the blacklist. They should be base names, not
 # paths.
-ignore=CVS,.git,embedded,venv
+ignore=CVS,.git,embedded,venv,.ropeproject
```

Comment:
I think these are created by https://github.com/python-rope/rope, I am not sure which of the tools we use calls it but I had a bunch of `.ropeproject` directories in a few of my integrations, and these in turn had a `config.py` that was confusing pylint when trying to lint _is_affirmative() which lives in dd-agent's `config.py`.